code/sql/schema.sql

```diff
@@ -1,96 +1,119 @@
-CREATE SCHEMA waze;
+CREATE SCHEMA IF NOT EXISTS  waze;
+
+CREATE TABLE waze.data_files
+(
+"id"                                SERIAL PRIMARY KEY NOT NULL,
+"start_time_millis"                 BIGINT NOT NULL,
+"end_time_millis"                   BIGINT NOT NULL,
+"start_time"                        TIMESTAMP,
+"end_time"                          TIMESTAMP,
+"date_created"                      TIMESTAMP,
+"date_updated"                      TIMESTAMP,
+"file_name"                         TEXT NOT NULL,
+"json_hash"                         VARCHAR(40) NOT NULL
```

Comment:
Shouldn't this field have a UNIQUE constraint?

Reply:
Yes, and in fact I've added that on mine but not pushed that change.  Here's my unique index, but I'm open to something different if necessary:
```CREATE UNIQUE INDEX "IDX_UNIQUE_json_hash"
ON waze.data_files USING btree
(json_hash COLLATE pg_catalog."default")
TABLESPACE pg_default;```

Reply:
@jrstill why not just VARCHAR(40) UNIQUE NOT NULL?

Reply:
I wanted an index because every time I process a file I have to first look up by hash (to see if we already have the record or not), and if found then decide whether to update the record (if the filename was the same) or throw an error with specifics about why we couldn't process the file.  I wanted to go ahead and put an index on it to keep that lookup fast.